wallet/storage/src/internal/mod.rs

```diff
@@ -161,7 +161,7 @@ impl<B: storage::Backend> Store<B> {
 
     /// Dump raw database contents
     pub fn dump_raw(&self) -> crate::Result<storage::raw::StorageContents<Schema>> {
-        self.storage.dump_raw().map_err(crate::Error::from)
+        self.storage.transaction_ro().unwrap().dump_raw().map_err(crate::Error::from)
```

Comment:
I think `?` could be used instead of unwrap here but may be wrong.

Reply:
Done